help/keyboard-shortcuts.md

```diff
@@ -81,13 +81,13 @@ in the Zulip app to add more to your repertoire as needed.
 
 * **Narrow to next unread topic**: <kbd>N</kbd>
 
-* **Narrow to next unread private message**: <kbd>P</kbd>
+* **Narrow to next unread direct message**: <kbd>P</kbd>
 
 * **Narrow to stream**: <kbd>S</kbd>
 
-* **Narrow to topic or PM conversation**: <kbd>Shift</kbd> + <kbd>S</kbd>
+* **Narrow to topic or DM conversation**: <kbd>Shift</kbd> + <kbd>S</kbd>
 
-* **Narrow to all private messages**: <kbd>Shift</kbd> + <kbd>P</kbd>
+* **Narrow to all direct messages**: <kbd>Shift</kbd> + <kbd>P</kbd>
```

Comment:
@alya just a note on an awkward detail with keyboard shortcuts: the `p/P` keyboard shortcuts are now not super intuitively named, and we already have a `d/D` shortcut. I don't think we have to do anything about this immediately, but wanted to flag it. Probably worth some discussion in the #design thread.

Reply:
Posted https://chat.zulip.org/#narrow/stream/101-design/topic/.22Private.22.20vs.20.22Direct.22/near/1514589 about this.